### DISCUSSION TITLE
Vertical Scroll?

### DISCUSSION THREAD

**Discussion by retr0h** [flant/grafana-statusmap#159] [Ideas] [answered]
I have some of the status maps panels broken down into smaller chunks.  However, we also have a section where we want to show all of our infra status at a very high level for historical purposes (more or less service uptime).  However, as the number of servers grow, the panel becomes very messy.

Query :

`((avg_over_time(up{job=~"node.*"}[75s]) * 75) > bool(60) or (avg_over_time(probe_success{job=~"blackbox.*"}[75s]) * 75) > bool(60)) * 100`

Screen shot of the output:

![Screen Shot 2020-05-28 at 2 48 31 PM](https://user-images.githubusercontent.com/9895/83197512-71343c80-a0f2-11ea-845c-e6af82e64f1e.png)

If we were to have vertical scrolling the issue would be resolved.  Running grafana 6.5.2.

**Comment by diafour** [comment, marked as answer]
At first thought, a vertical scroll is possible, and 'historical purposes' is a good use case. I need to check if render can be modified to support this.

p.s. Also, I have a strong feeling that the vertical scroll is not a good solution and should be avoided. In an ideal world, the dashboard should help the user to see that something is wrong with a brief look. The user should interact with panels only to dive into details of a 'bad state' and not to search for entities in a 'bad state'.

Maybe something from this list is more suitable for your case?
- There is a pagination approach to this problem in #93, it is discussed in #83. 
- There is an issue #26 which proposes to support groups and hierarchy for rows.
- More sorting options are proposed in #30.
